src/function/localStorage.test.js

```diff
@@ -0,0 +1,51 @@
+import StoreLists from './localStorage.js';
+import TestList from './testList.js';
+
+describe('should check the item is added', () => {
+  test('should add to the list', () => {
+    const test = new TestList(1, 1, 'sleep', false);
+    const testlist = StoreLists.add(test);
+    expect(testlist).toHaveLength(1);
+  });
+
+  test('should add to the list', () => {
+    const test = new TestList(2, 2, 'finish the project', false);
+    const testlist = StoreLists.add(test);
+    expect(testlist).toHaveLength(2);
+  });
+});
+
+// describe('should delete from the array', () => {
+//   test('should delete from the list', () => {
+//     StoreLists.remove(2);
+//     expect(StoreLists.remove(2)).toHaveLength(1);
+//   });
+//   test('should delete from the list', () => {
+//     StoreLists.remove(1);
+//     expect(StoreLists.remove(1)).toHaveLength(0);
+//   });
+// });
```

Comment:
- [ ] Kindly do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. Please remove that.

Reply:
Thanks for your suggesting,  I commented out and run as a function now.